debian/jessie/kubectl

```diff
@@ -1 +0,0 @@
-../bionic/kubectl
```

Comment:
Can you clarify why this is deleted?

Reply:
@sumitranr -- previously, only the kubectl definitions were symlinked. This PR symlinks the full directory, so all binaries are built for each deb distro flavor.

See here: https://github.com/kubernetes/release/pull/720/files#diff-e3f51e32c56fc68e9e81fd19c758c774

Reply:
ah! i should have read your change description more thoroughly. you have already described it there. thanks for clarifying.